chromiumoxide_pdl/src/build/generator.rs

```diff
@@ -1007,14 +1007,12 @@ pub(crate) fn generate_field_name(name: &str) -> String {
     }
 }
 
-/// Escapes reserved rust keywords
-pub(crate) fn generate_type_name(name: &str) -> String {
-    let name = name.to_upper_camel_case();
```

Comment:
This was some non-sense copy paste from the above method. "to upper camelcase" -> "lowercase match"